docs/core/tools/telemetry.md

```diff
@@ -26,17 +26,14 @@ Telemetry *is collected* when using any of the [.NET CLI commands](index.md), su
 
 The .NET SDK telemetry feature is enabled by default for Microsoft distributions of the SDK. To opt out of the telemetry feature, set the `DOTNET_CLI_TELEMETRY_OPTOUT` environment variable to `1` or `true`.
 
-A single telemetry entry is also sent by the .NET SDK installer when a successful installation happens. To opt out, set the `DOTNET_CLI_TELEMETRY_OPTOUT` environment variable before you install the .NET SDK.
+The .NET SDK installer sends a single telemetry entry when a successful installation happens. To opt out, set the `DOTNET_CLI_TELEMETRY_OPTOUT` environment variable before you install the .NET SDK.
 
 > [!IMPORTANT]
-> To opt out after you started the installer: close the installer, set the environment variable, and then run the installer again with that value set.
+> To opt out after you start the installer: close the installer, set the environment variable, and then run the installer again with that value set.
 
 ## Disclosure
 
-The .NET SDK displays text similar to the following when you first run one of the [.NET CLI commands](index.md) (for example, `dotnet build`). Text might vary slightly depending on the version of the SDK you're running. This "first run" experience is how Microsoft notifies you about data collection.
-
-> [!NOTE]
-> **Breaking change:** The behavior of telemetry messages written to `stderr` has changed in recent versions of the .NET SDK. For more information, see [dotnet CLI commands log non-command-relevant data to stderr](../compatibility/sdk/10.0/dotnet-cli-stderr-output.md).
```

Comment:
Do you think it's good to remove this considering this is new behavior introduced in .NET 10?